dailyperf/tests/icon_border/test.md

```diff
@@ -13,28 +13,27 @@ _Between border and icon, witch one impact the most the performance?_
 |  N    |  Y    |  N   |   N   |
```

Comment:
R: No impact on responsive time
A: Minor Impact difference if you render element during an animation
I: N/A
L: No impact on the load time (we exclude the impact of loading the webfont)